### ISSUE TITLE
Allow the account to be modified

### ISSUE THREAD

**Issue by dmcgrath** [hepwori/hepwori.github.io#1]
I'd really like to have "Twitter Surveys" as the account. It would be nice to not be stuck with just Twitter.

/firstworldproblems

**Comment by hepwori** [comment]
Yeah doing a generic one of these (where you can select the app which notifies) is on my list.
